includes/templates/template_default/templates/tpl_index_default.php

```diff
@@ -68,6 +72,15 @@
 ?>
 <?php include(DIR_WS_MODULES . zen_get_module_directory(FILENAME_UPCOMING_PRODUCTS)); ?><?php } ?>
 
+<?php if ($show_display_category->fields['configuration_key'] == 'SHOW_PRODUCT_INFO_MAIN_FEATURED_CATEGORIES') { ?>
+<?php
+/**
+ * display the Featured Products Center Box
+ */
+?>
+<?php require($template->get_template_dir('tpl_modules_featured_categories.php',DIR_WS_TEMPLATE, $current_page_base,'templates'). '/tpl_modules_featured_categories.php'); ?>
```

Comment:
- Remove leading/trailing parentheses around required file.
- Add single-space between function arguments and after function (i.e. before the concatenating `.`).